observation_portal/requestgroups/serializers.py

```diff
@@ -24,42 +25,92 @@
 from observation_portal.common.configdb import configdb, ConfigDB, ConfigDBException
 from observation_portal.requestgroups.duration_utils import (
     get_request_duration, get_request_duration_sum, get_total_duration_dict, OVERHEAD_ALLOWANCE,
-    get_instrument_configuration_duration, get_num_exposures, get_semester_in
+    get_instrument_configuration_duration, get_semester_in
 )
 from datetime import timedelta
 from observation_portal.common.rise_set_utils import get_filtered_rise_set_intervals_by_site, get_largest_interval
 
 logger = logging.getLogger(__name__)
 
 
-def cerberus_validation_error_to_str(validation_errors: dict) -> str:
-    error_str = ''
-    for field, value in validation_errors.items():
-        if (isinstance(value, list) and len(value) == 1 and isinstance(value[0], dict)):
-            error_str += f'{field}{{{cerberus_validation_error_to_str(value[0])}}}'
-        else:
-            error_str += f'{field} error: {", ".join(value)}, '
+class ValidationHelper(ABC):
+    """Base class for validating configurations"""
```

Comment:
The word "configurations" here means some dictionary with fields to validate, not necessarily a configuration in terms of the Configurations of a Request?

Reply:
Good point. It's not necessarily a Configuration from a Request. Presumably it could be any structure that needs to be validated via Cerberus.

Cerberus' docs use the term "document" a lot -  which seems more appropriate?

Reply:
Yeah I think "document" sounds good! Definitely would make it more clear.